schemas/Semantic.schema.json

```diff
@@ -890,10 +890,10 @@
                 {
                     "type": "string",
                     "const": "UNKNOWN_ACTIVITY_TYPE",
-                    "title": "Moving",
-                    "description": "",
+                    "title": "Uknown Activity",
+                    "description": "Type used when unable to detect the current activity.",
                     "extra_color": "#03a9f4",
-                    "extra_icon": "activity_types/moving_24dp.svg"
+                    "extra_icon": "activity_types/unknown_24dp.svg"
```

Comment:
I didn't realize that the icons are from https://github.com/CarlosBergillos/LocationHistoryFormat/tree/main/docs/static/icons/activity_types and used in the generated docs

![Screenshot 2024-03-20 231425](https://github.com/CarlosBergillos/LocationHistoryFormat/assets/99822/31dab1f2-8a28-4559-83c5-81940feba8f5)